app/views/incident_reports/_form.html.haml

```diff
@@ -244,6 +244,32 @@
         prompt: true
   .passenger-incident-info{style: ('display: none;' unless report.passenger_incident?)}
     %h3 Passenger Incident Information
+    .field
```

Comment:
A nitpick, but the placement of this is a bit weird to me. I would expect the accordion to be exactly below the `Did the incident involve a passenger?`.

Reply:
Agreed. It was based on the PDF form, but for a web form the layout isn't intuitive. Opened https://github.com/umts/incidents/issues/134.